pkg/integration/tests/custom_commands/form_prompts.go

```diff
@@ -0,0 +1,88 @@
+package custom_commands
+
+import (
+	"github.com/jesseduffield/lazygit/pkg/config"
+	. "github.com/jesseduffield/lazygit/pkg/integration/components"
+)
+
+var FormPrompts = NewIntegrationTest(NewIntegrationTestArgs{
+	Description:  "Using a custom command reffering prompt responses by name",
+	ExtraCmdArgs: "",
+	Skip:         false,
+	SetupRepo: func(shell *Shell) {
+		shell.EmptyCommit("blah")
+	},
+	SetupConfig: func(cfg *config.AppConfig) {
+		cfg.UserConfig.CustomCommands = []config.CustomCommand{
+			{
+				Key:     "a",
+				Context: "files",
+				Command: `echo "{{index .Form.FileContent}}" > {{index .Form.FileName}}`,
```

Comment:
is the `index` thing here necessary?

Reply:
It looks like I forgot to remove it. I updated the code now